pkg/controller/template/test_data/controller_config_ovirt.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "machineconfigurations.openshift.io/v1"
+kind: "ControllerConfig"
+spec:
+  clusterDNSIP: "10.3.0.10"
+  cloudProviderConfig: |
+    testing
+    multi-line cloud config
+    [test]
+      option = dummy
+  etcdDiscoveryDomain: "my-test-cluster.metalkube.org"
```

Comment:
Looks like `metalkube.org` is an owned domain. Just checking this is meant to be used as a default/dummy instead of `my-test-cluster.installer.team.coreos.systems`.